packages/client/package.json

```diff
@@ -9,6 +9,7 @@
   "private": true,
   "dependencies": {
     "@types/react-flatpickr": "^3.7.1",
+    "@types/react-tooltip": "^3.9.2",
```

Comment:
Types should be installed with the -D flag and belong in the devDependencies section.